corehq/ex-submodules/couchforms/tests/test_meta.py

```diff
@@ -176,3 +172,11 @@ def testMetaDateInDatetimeFields(self):
 
         self.assertEqual(datetime(2014, 7, 10), xform.metadata.timeStart)
         self.assertEqual(datetime(2014, 7, 11), xform.metadata.timeEnd)
+
+    @run_with_all_backends
+    def test_missing_meta_key(self):
```

Comment:
it was based on this comment: https://github.com/dimagi/commcare-hq/pull/8964#discussion_r44109703
tests still passed, but they should have failed